tests/test_configuration.py

```diff
@@ -0,0 +1,81 @@
+import json
+from os import environ
+from unittest.mock import MagicMock, patch
+
+from lbz.aws_boto3 import SSM
+from lbz.configuration import BaseValue, EnvValue, SSMValue
+
+
+class TestBaseConfig:
+    @patch.object(BaseValue, "getter", autospec=True)
+    def test_if_getter_was_used(self, mocked_getter: MagicMock) -> None:
+        cfg = BaseValue("key")  # type: ignore
```

Comment:
I didn't check the tests yet but definitely, you cannot do things like this.

`BaseValue` is an abstract class so if you want to rely on this you need to create a new one 😉